.github/workflows/prismic-linting.yml

```diff
@@ -0,0 +1,25 @@
+name: Prismic Linting
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 8 * * *'
```

Comment:
We could also restrict it to office days? I think it'd be
`0 8 * * 1-5`
or for maybe better readability
`0 8 * * MON-FRI`